skins/cat17/src/app/lib/scrolling.js

```diff
@@ -22,9 +22,14 @@ var objectAssign = require( 'object-assign' ),
 
 	AnimatedScroller = {
 		fixedHeaderElements: null,
-		scrollTo: function( $element ) {
+		scrollTo: function( $element, options ) {
+			var topOffset = calculateElementOffset( $element, this.fixedHeaderElements );
+			if ( options && options.additionalOffset ) {
```

Comment:
I thought this class was to encapsulate the offset calculation. Which circumstances led to it now requiring an external information instead, which would not better be _calculated_ (instead of hard coded) by this class (instead of somewhere else)?

Reply:
`calculateElementOffset` removes the padding of the elements it scrolls to, which is important for the scrolls from the info sections (as the form sections have a big padding above them). The info box has a border and a padding, hence the padding needs to be re-added. 
Possible solutions:

* fiddle with the CSS/Markup (of the form sections, not the infobox) to find better scroll points, dropping the subtraction of padding.
* instead of adding explicit offsets, have a `options.subtractPadding` boolean flag that is given to `calculateElementOffset`
* ???

What do you think?